src/raygun.breadcrumbs.js

```diff
@@ -0,0 +1,296 @@
+/*
+ * raygun4js
+ * https://github.com/MindscapeHQ/raygun4js
+ *
+ * Copyright (c) 2017 MindscapeHQ
+ * Licensed under the MIT license.
+ */
+/* globals console */
+
+var raygunBreadcrumbsFactory = function(window, $, Raygun) {
```

Comment:
We generally have been wrapping these methods in self-executing functions to limit the amount of global variables we add. Unless there is a reason we might need to create a second Breadcrumb?